Cubical/Categories/Displayed/Presheaf/Constructions/Reindex/Properties.agda

```diff
@@ -241,6 +232,35 @@ module _ {C : Category ℓC ℓC'}{Cᴰ : Categoryᴰ C ℓCᴰ ℓCᴰ'}
             })
     (λ a → record { equiv-proof = strictContrFibers _ })
 
+module _ {C : Category ℓC ℓC'}{Cᴰ : Categoryᴰ C ℓCᴰ ℓCᴰ'}
+  {P : Presheaf C ℓP}{Q : Presheaf C ℓQ}
+  {α : PshIso P Q}
+  {Pᴰ : Presheafᴰ P Cᴰ ℓPᴰ}
+  {Qᴰ : Presheafᴰ Q Cᴰ ℓQᴰ}
+  (αᴰ : PshIsoᴰ α Pᴰ Qᴰ)
+  where
+
+  private
+    module Pᴰ = PresheafᴰNotation Pᴰ
+    module Qᴰ = PresheafᴰNotation Qᴰ
+
+  PshIsoᴰ→PshIsoⱽ : PshIsoⱽ Pᴰ (reind (α .trans) Qᴰ)
```

Comment:
This should not be implemented manually. This can be defined using `reind-introⱽ` and `reind-π`.